README.md

```diff
@@ -1 +1,216 @@
+
+- [📖 Appointments App ](#-appointments-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [Kanban Board ](#kanban-board-)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Appointments App <a name="about-project"></a>
+
+This is the Appointments App, a web application built with Ruby on Rails, PostgreSQL, JavaScript, React, and Redux. It allows users to manage appointments and schedule appointments with engineers.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+</details>
+<details>
+  <summary>Database</summary>
+    <li><a href="https://www.postgresql.org/">Postgres</a></li>
+</details>
+<details>
+  <summary>Client</summary>
+    <li><a href="https://www.javascript.com/">Javascript</a></li>
+    <li><a href="https://reactjs.org/">React</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```
+    - ruby >= 3.2.2
+    - rails >= 7.1.3
+    - postgres >= 14.7
+    - bundler 2.5.1
+    - node.js 19.6.0
+    - yarn 3.4.1
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash
+  git clone https://github.com/MohammadYaser/appointment-app.git
+```
+
+### Install
+
+Install this project with:
+
+```bash
+  cd appointments_app
+  bundle install
+  yarn install
+```
+
+You will need to follow these steps to have your own set of related project secrets:
+
+- delete `config/credentials.yml.enc` file
+- use this command `EDITOR="code --wait" bin/rails credentials:edit`
+- under `secret_base_key` add this line `devise_jwt_secret_key:`
+- in another terminal run `bundle exec rails secret`
+- take the output and use it as value for `devise_jwt_secret_key`
+- save the file and close it
+
+it will install the required gemfile for running the project
+
+run the following commands to create and migrate:
+
+```bash
+  rails db:create
+  rails db:migrate
+  rails db:seed
+```
+
+### Usage
+
+to use this project:
+
+```sh
+   bin/dev
+```
+
+it will run the the server on `localhost:3000`
+
+to run tests:
+
+```sh
+   rspec --format doc
+```
```

Comment:
- [ ] When I run the backend tests according to the instructions, I get this:

![image](https://github.com/MohammadYaser/appointment-app/assets/57421823/2f6a856e-c384-4525-9e5c-7a42cf7c5923)

Kindly, fix these backend tests so your app's integrity is secured. 👍

Reply:
Hello Meltrust,

we ran rspec and found two errors from the sessions_spec file only.
The consultations_spec and users_spec probably fail because there was no data in the database. 
There was an issue with the seeds file and how user creation was handled which has been fixed.